ReCaptchaFrontendUi/Plugin/ExcludeFromMinification.php

```diff
@@ -1,33 +0,0 @@
-<?php
```

Comment:
Why was this deleted?

Reply:
https://*/recaptcha/api.js is not catchable by the Magento minifier as the url is set dynamically in js

Reply:
@nathanjosiah

Reply:
My concern is that this will break existing expectations for minification exclusion. Are you saying this was not working already before your changes? As far as I can see the URL was static before you added the new URL.

Reply:
I'm not sure to understand, but the only static url in the template was ReCaptchaUser/view/adminhtml/templates/recaptcha.phtml and it's removed with this PR. So I thought it's safe to remove it from the minification exclusion list.